client/src/layout/Header/index.js

```diff
@@ -19,37 +25,40 @@ const links = [
 ];
 
 const Header = () => {
-  const {
-    isAuthenticated,
-    error,
-    user,
-    loginWithRedirect,
-    logout,
-  } = useAuth0();
+  const { isAuthenticated, user, loginWithRedirect, logout } = useAuth0();
 
   // @TODO: Change window.location to react router
   const handleLogout = () => logout({ returnTo: window.location.origin });
 
   return (
-    <header className={styles.header}>
-      <p className={styles.logo}>HackShare</p>
-      <div className={styles.spacer}></div>
-      {links.map((link, i) => (
-        <Link className={styles.link} to={link.route} key={i}>
-          {link.name}
-        </Link>
-      ))}
-      <button
-        className={styles.login}
-        onClick={isAuthenticated ? handleLogout : loginWithRedirect}
-      >
-        {isAuthenticated ? "Log out" : "Log in"}
-      </button>
-      {error && (
-        <div>Oops, Somethign went wrong... {JSON.stringify(error)}</div>
-      )}
-      {user && user.name}
-    </header>
+    <AppBar position="static" className={styles.header}>
+      <Toolbar>
+        <img className={styles.logo} src="/Logo.svg"></img>
```

Comment:
importing `svg` in `img` tags is not a good idea
Use `<Logo className={classes.logo} />` and import it by `import { ReactComponent as Logo} from "../../...." `;